docs/windows.md

```diff
@@ -49,15 +42,8 @@ The following components should be configured and run on the Windows Node.
 * [kubernetes components](https://kubernetes.io/docs/setup/production-environment/windows/user-guide-windows-nodes/)
 * OVS daemons
 * antrea-agent
-* kube-proxy
 
-antrea-agent and kube-proxy run as processes on host and are managed by
-management Pods. It is recommended to run OVS daemons as Windows services.
-We also support running OVS processes inside a container. If you don't want to
-run antrea-agent and kube-proxy from the management Pods Antrea also provides
-scripts which help to install and run these two components directly without Pod.
-Please see [Manually run kube-proxy and antrea-agent on Windows worker Nodes](#manually-run-kube-proxy-and-antrea-agent-on-windows-worker-nodes)
-section for details.
+antrea-agent and the OVS daemons can either run as Pods or as Windows services.
```

Comment:
Maybe some more clarification here ? If antrea-agent is running as service, OVS daemons also need to be run as service ?

Reply:
Done, I added a table which hopefully is useful